modules/Bio/EnsEMBL/Compara/PipeConfig/BuildNewMasterDatabase_conf.pm

```diff
@@ -164,6 +164,15 @@ sub pipeline_analyses {
                 'schema_file'   => $self->o('schema_file'),
                 'cmd'           => 'db_cmd.pl -reg_conf #init_reg_conf# -reg_type compara -reg_alias #master_db# < #schema_file#',
             },
+            -flow_into  => ['add_division_to_meta_table'],
+        },
+
+        {   -logic_name => 'add_division_to_meta_table',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+            -parameters => {
+                'init_reg_conf' => $self->o('init_reg_conf'),
+                'cmd'           => 'echo "INSERT INTO meta (species_id, meta_key, meta_value) VALUES (NULL, \"division\", \"#division#\");" | db_cmd.pl -reg_conf #init_reg_conf# -reg_type compara -reg_alias #master_db# < /dev/stdin'
```

Comment:
Do you need `< /dev/stin` ? Does the pipe not work ?
Otherwise, another way of doing that is `['db_cmd.pl', ..., -sql => '#sql_insert#']` and define the `sql_insert` parameter separately. Then you don't need to worry about quotes and escaping them.

Reply:
I couldn't find how to do it the way you propose, so `< /dev/stin` was an acceptable solution.

Reply:
I was thinking of this:
```perl
'sql' => 'INSERT INTO meta (species_id, meta_key, meta_value) VALUES (NULL, "division", "#division#")' ,
'cmd' => ['db_cmd.pl', -reg_conf => '#init_reg_conf#', -reg_type => 'compara', -reg_alias => '#master_db#', sql => '#sql#'],
```